vmaas/vulnerabilities.go

```diff
@@ -359,15 +362,15 @@ func evaluateTest(c *Cache, testID TestID, pkgNameID NameID, nevra utils.Nevra)
 	return matched
 }
 
-func updateCves(cves map[string]VulnerabilityDetail, cve, pkg string, erratum *string) {
+func updateCves(cves map[string]VulnerabilityDetail, cve, pkg string, erratum []string) {
 	if _, has := cves[cve]; !has {
 		cveDetail := VulnerabilityDetail{
 			CVE:      cve,
 			Packages: []string{pkg},
 			Errata:   []string{},
 		}
 		if erratum != nil {
-			cveDetail.Errata = []string{*erratum}
+			cveDetail.Errata = erratum
```

Comment:
I think you can remove the `if erratum != nil` and add `erratum` directly to `VulnerabilityDetail` struct literal